lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -55,131 +59,102 @@ public function init() {
-        </span>
-      </div>
-    <?php
+  public function render_sso( $args = array() ) {
+    $this->render_switch( $args['label_for'], $args['opt_name'] );
+    $this->render_field_description(
+      __( 'SSO allows users to sign in once to multiple Clients in the same tenant. ', 'wp-auth0' ) .
+      __( 'Turning this on will attempt to automatically log a user in when they visit wp-login.php. ', 'wp-auth0' ) .
+      __( 'This setting will not affect how shortcodes and widgets work. ', 'wp-auth0' ) .
+      __( 'For more information, see our ', 'wp-auth0' ) .
+      $this->get_docs_link( 'sso/current/introduction', __( 'help page on SSO', 'wp-auth0' ) )
+    );
   }
 
-  public function render_singlelogout() {
-    $v = absint( $this->options->get( 'singlelogout' ) );
-
-    echo $this->render_a0_switch( "wpa0_singlelogout", "singlelogout", 1, 1 == $v );
-?>
-
-      <div class="subelement">
-        <span class="description">
-          <?php echo __( 'Single Logout is the opposite of the above SSO, it logs users out of everything at once. For more details, see our', 'wp-auth0' ); ?>
-          <a target="_blank" href="https://auth0.com/docs/sso/single-sign-on"><?php echo __( 'help page on SSO', 'wp-auth0' ); ?></a>.
-        </span>
-      </div>
-    <?php
+  public function render_singlelogout( $args = array() ) {
+    $this->render_switch( $args['label_for'], $args['opt_name'] );
+    $this->render_field_description(
+      __( 'Log users out of this site and all others connected to the tenant', 'wp-auth0' )
+    );
   }
 
-  public function render_mfa() {
-    $v = $this->options->get( 'mfa' );
-
-    echo $this->render_a0_switch( "wpa0_mfa", "mfa", 1, !empty( $v ) );
-?>
-
-      <div class="subelement">
-        <span class="description">
-          <?php echo __( 'Mark this if you want to enable multifactor authentication with Auth0 Guardian. For more information, see ', 'wp-auth0' ); ?>
-          <a target="_blank" href="https://auth0.com/docs/mfa"><?php echo __( 'our help page on MFA', 'wp-auth0' ); ?></a>.
-          <?php echo __( 'You can enable other MFA providers from the ', 'wp-auth0' ); ?>
-          <a target="_blank" href="https://manage.auth0.com/#/multifactor"><?php echo __( 'Auth0 dashboard', 'wp-auth0' ); ?></a>.
-          <?php echo __( 'You can reset your users MFA provider data, by going to the user and clicking on "Delete MFA Provider" button.', 'wp-auth0' ); ?>
-        </span>
-      </div>
-    <?php
+  public function render_mfa( $args = array() ) {
+    $this->render_switch( $args['label_for'], $args['opt_name'] );
+    $this->render_field_description(
+      __( 'Mark this if you want to enable multifactor authentication with Auth0 Guardian. ', 'wp-auth0' ) .
+      sprintf(
+        __( 'You can enable other MFA providers in the %s. ', 'wp-auth0' ),
+        $this->get_dashboard_link( 'multifactor' )
+      ) . __( 'For more information, see our %s. ', 'wp-auth0' ) .
```

Comment:
This `%s` is not inside the `sprintf` scope so I think there's no substitution happening here, am I right?